### PR TITLE
architecture-overview

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Architecture overview
 
-Nebula Graph consists of three services: the Graph Service, the Storage Service, and the Meta Service.
+Nebula Graph consists of three services: the Graph Service, the Storage Service, and the Meta Service. It applies a disaggregated storage and computing architecture.
```
Comment (whitewum) [inline docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md:3]: the separation of computing and storage architecture.

Comment (whitewum) [reply docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md:3]: it is a tech term

File: docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md [hunk 2]
```diff
@@ -16,11 +16,22 @@ For details on the Meta Service, see [Meta Service](2.meta-service.md).
 
 ## The Graph Service and the Storage Service
 
-Nebula Graph applies a disaggregated storage and compute architecture. The Graph Service is responsible for querying. The Storage Service is responsible for storage. And they run on different processes, i.e., nebula-graphd and nebula-storaged. The benefits of disaggregated storage and compute are as follows:
+Nebula Graph applies a disaggregated storage and computing architecture. The Graph Service is responsible for querying. The Storage Service is responsible for storage. They are run by different processes, i.e., nebula-graphd and nebula-storaged. The benefits of a disaggregated storage and computing architecture are as follows:
```
Comment (whitewum) [inline docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md:19]: ditto

File: docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md [hunk 3]
```diff
@@ -16,11 +16,22 @@ For details on the Meta Service, see [Meta Service](2.meta-service.md).
...
 
-* Great scalability. A disaggregated structure makes both the Graph Service and the Storage Service flexible and easy to scale in or out.
-* High availability. If part of the Graph Service fails, the data stored by the Storage Service suffers no loss. And if the rest part of the Graph Service is still able to serve the clients, service recovery can be performed quickly, or even unfelt by the users.
-* Cost-effective. The separation of computing and storage provides a higher resource utilization rate, and it enables you to manage the cost flexibly according to business demands. The cost savings can be more significant if you use the [Nebula Graph Cloud](https://www.nebula-cloud.io/ "Nebula Graph Cloud official website") service.
-* Open to more possibilities. With the ability to run separately, the Graph Service may work with multiple types of storage engines, and the Storage Service may serve more types of computing engines.
+* Great scalability
+
+  A disaggregated structure makes both the Graph Service and the Storage Service flexible and easy to scale in or out.
```
Comment (whitewum) [inline docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md:23]: ditto

**Review by whitewum — COMMENTED** [review]
minor

**Review by whitewum — APPROVED** [review]

File: docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md [hunk 4]
```diff
@@ -16,11 +16,22 @@ For details on the Meta Service, see [Meta Service](2.meta-service.md).
...
+
+  The separated structure makes both the Graph Service and the Storage Service flexible and easy to scale in or out.
+
+* High availability
+
+  If part of the Graph Service fails, the data stored by the Storage Service suffers no loss. And if the rest part of the Graph Service is still able to serve the clients, service recovery can be performed quickly, even unfelt by the users.
+
+* Cost-effective
+  
+  The separation of storage and computing architecture provides a higher resource utilization rate, and it enables clients to manage the cost flexibly according to business demands. The cost savings can be more distinct if the [Nebula Graph Cloud](https://www.nebula-cloud.io/ "Nebula Graph Cloud official website") service is used.
```
Comment (randomJoe211) [inline docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md:31]: Optimize the first sentence in the next PR.

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**